en/contents.rst

```diff
@@ -57,6 +57,7 @@ Contents
     core-libraries/xml
 
     core-libraries/global-constants-and-functions
+    debug-kit
```

Comment:
I thought about that too, but I feel like as we add more of the plugins having them grouped together outside of the core docs might be nice.